src/main/java/org/isf/admission/gui/PatientFolderBrowser.java

```diff
@@ -885,43 +851,41 @@ public LabBrowserModel() {
 			}
 		}
 		
+		@Override
 		public int getRowCount() {
 			if (labList == null)
 				return 0;
 			return labList.size();
 		}
 
+		@Override
 		public String getColumnName(int c) {
 			return plColumns[c];
 		}
 
+		@Override
 		public int getColumnCount() {
 			return plColumns.length;
 		}	
 		
 		
-		public Object getValueAt(int r, int c) {
-			if (c == -1) {
-				return labList.get(r);
-			} else if (c == 0) {
-				//System.out.println(labList.get(r).getExam().getExamtype().getDescription());
-				
-				Date examDate = labList.get(r).getExamDate().getTime();	
+		@Override
+		public Object getValueAt(int row, int column) {
+			if (column == -1) {
```

Comment:
here we can start to put `labList.get(row)` as common variable at the top

Reply:
Done